Week-1/cattyBoard/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CattyBoard</title>
+    <link rel="stylesheet" href="index.css" />
+    <script
+      src="https://kit.fontawesome.com/ff867e86d4.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main class="main">
+      <nav id="nav">
+        <i class="fa-solid fa-bars"></i>
+        <i class="fa-solid fa-house-chimney"></i>
+        <i class="fa-solid fa-magnifying-glass"></i>
+        <i class="fa-solid fa-volume-high"></i>
+        <i class="fa-solid fa-user"></i>
+        <i class="fa-brands fa-spotify"></i>
+        <i class="fa-solid fa-gear"></i>
+        <i class="fa-brands fa-soundcloud"></i>
+      </nav>
+      <section>
+        <header>
+          <div>
+            <img
+              src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"
+              alt=""
+              srcset=""
+            />
+          </div>
+          <div id="songName">
+            <h1>CattyBoard Top 100 Single Charts(11.06.36)</h1>
+            Unknown Artist <br />
+            2016.Charts.100songs
+            <p>
+              <span><i class="fa-solid fa-play">&nbsp&nbspPlay all</i></span>
+              <span><i class="fa-solid fa-plus">&nbsp&nbspAdd to</i></span>
+              <span><i class="fa-solid fa-ellipsis">&nbsp&nbspMore</i></span>
+            </p>
+          </div>
+        </header>
+        <article>
+          <table>
```

Comment:
After you are comfortable with basic layout and css, I would recommend going through the flex and/or grid. 
The layout you created with table can also be created using flex and using flex instead of tables is very common as it provides cross browser compatibility and responsive ui support. 
Would suggest you to go through why tables are not very commonly used in the industry.